packages/satusehat/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# @openfn/language-template
```

Comment:
Did you generate this with the tempate CLI? If so, we need to fix the changelog!

Reply:
I have changed it

Reply:
Can we fix the template too? It's fine to do it in this PR

Reply:
@josephjclark  I have fixed the template as well